integreat_cms/cms/models/external_calendars/external_calendar.py

```diff
@@ -31,6 +31,15 @@ class ExternalCalendar(AbstractBaseModel):
         ),
     )
     errors = models.CharField(verbose_name=_("import errors"), default="", blank=True)
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        null=True,
+        blank=True,
```

Comment:
Hmm, I assume thi is because external calendars have already been created without an user?
"created_by" seems like a field that should be mandatory - do you think it woul be feasible to set a default user for the existing entries in the migration?

Reply:
I thought so too initially, but that would cause a problem when a user account gets removed (right now the code can simply say `SET_NULL`)

Reply:
Hm, true. I guess the only way around this would be to use `SET_DEFAULT` and use the same default user as for the migration. But yeah, not sure if worth it